content/reference/user/artifact_gen/index.md

```diff
@@ -1,33 +1,36 @@
 ---
 title: "Artifact Generation"
-description: "Learn how to generate an artifact for FrontLine from Gatling zip bundle or a maven, SBT or gradle project."
-lead: "Generate an artifact from your Gatling bundle or with a maven, SBT or gradle project."
+description: "Learn how to generate an artifact for Gatling Enterprise from the Gatling zip bundle, or from a Maven, SBT or Gradle project."
```

Comment:
```suggestion
description: "Learn how to generate an artifact for Gatling Enterprise from the Gatling zip bundle, or from a Maven, SBT, or Gradle project."
```

Reply:
Oh, the good old Oxford comma debate. 😅 Let's go for it, it would be useful in more complex sentences and it's usually not recommended to switch styles.